cli/src/pcluster/aws/aws_resources.py

```diff
@@ -288,22 +288,28 @@ def ec2memory_size_in_mib(self):
         return self.instance_type_data.get("MemoryInfo", {}).get("SizeInMiB")
 
 
-class FsxFileSystemInfo:
-    """Data object wrapping the result of a describe_file_systems call."""
+class FsxStorageInfo:
+    """Data object wrapping the result of a describe_file_systems and describe_file_caches call."""
 
-    def __init__(self, file_system_data):
-        self.file_system_data = file_system_data
+    def __init__(self, file_storage_data):
+        self.file_storage_data = file_storage_data
```

Comment:
I think you're renaming `file_system_data` into `file_storage_data` to make this logic more generic to cover both file systems and file caches.  Nevertheless, the concept of `file_storage` is really uncommon. So I would suggest to rename it in `storage_info`. because it contains the information about the storage returned by FSx.

what do you think?

Reply:
Sure

Reply:
Will change it to file_storage_info